examples/100streams.js

```diff
@@ -59,92 +35,31 @@ th.run({
       // these are the livepeer binary flags, add them as you wish.
       // the test-harness overrides flags that has to do with directories or
       // ip/port bindings, these are automated.
-      flags: '--v 4 -transcoder -initializeRound=true -standaloneTranscoder=true \
+      flags: '-v 5 -transcoder -initializeRound=true -standaloneTranscoder=true \
         -orchAddr https://orchestrator_0:8935 -orchSecret test'
     },
     orchestrators: {
-      instances: 25,
+      instances: 15,
       // TODO these are not complete, try adding the right orchestrator flags :)
-      flags: `--v 4 -initializeRound=true \
-      -gasPrice 200 -gasLimit 2000000 \
-      -monitor=false -currentManifest=true -transcoder`
+      flags: `-v 5 -initializeRound=true -gasPrice 20 -gasLimit 20000000 \
+      -currentManifest=true -maxSessions 8`
```

Comment:
Note that Os technically don't need the `currentManifest` flag since they don't serve manifests. Harmless to have it here, but it's not strictly necessary either.